app/schemas/org.p2p.wallet.infrastructure.db.WalletDatabase/4.json

```diff
@@ -0,0 +1,582 @@
+{
```

Comment:
this file will help us to prevent crashes when we change database but forget to bump its version

Reply:
Well that file helped me to check that I did everything ok with primary keys and all entities. 
Maybe you're right, we don't need that for now. 
But! How should we save user from deauth when bumping DB version?

Reply:
User data is stored in prefs. 
Database has only info about his tokens that are being fetched everytime he is signing in.
And history which you made. 

Thus, bumping DB version doesn't effect user's experience

Reply:
ah okay, i'll remove it then